### PR TITLE
fix(@vtmn/svelte, @vtmn/react, @vtmn/vue): `VtmnNavbar` slots

### PR DESCRIPTION
**Pull request by Mig-l**
## Changes description

1- VtmnNavBar

The logo link should not be part of the component but rather come from the slot.
Currently, we cannot track the click on the logo, etc...

2- VtmnNavBarLink

Events on the a tag do not go back to the parent.
$$restProps does not forward events, you have to add them.
I refactored in relation to the "showLabel" variable and I added an "if" around the icon

## Checklist

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch**. Please, don't request directly from your main!
- [x] Check commits & PR names matches our requested structure. It must follow the https://www.conventionalcommits.org pattern.
- [ ] Check your code additions will fail neither code linting checks.
- [ ] I have reviewed the submitted code.
- [ ] I have tested on related showcases.
- [ ] If it includes design changes, please ask for a review `design-system-core-team-design` GitHub team.

## Does this introduce a breaking change?

- Yes

### CODE REVIEW

File: packages/sources/svelte/src/components/navigation/VtmnNavbarLink/VtmnNavbarLink.svelte [hunk 1]
```diff
@@ -43,19 +43,24 @@
...
-{:else}
-  <a class={componentClass} {...$$restProps}>
-    <slot />
-    <VtmnIcon value={icon} aria-hidden="true" />
-    <span class="vtmn-sr-only">{label}</span>
-  </a>
-{/if}
+  {/if}
+  <span class:vtmn-sr-only={!showLabel}>{label}</span>
+</a>
```
Comment (Tlahey) [inline packages/sources/svelte/src/components/navigation/VtmnNavbarLink/VtmnNavbarLink.svelte:63]: Can be rollback, this part is fixed on this PR https://github.com/Decathlon/vitamin-web/pull/1425

Comment (Mig-l) [reply packages/sources/svelte/src/components/navigation/VtmnNavbarLink/VtmnNavbarLink.svelte]: Done

File: packages/sources/svelte/src/components/navigation/VtmnNavbar/VtmnNavbar.svelte [hunk 2]
```diff
@@ -39,9 +32,9 @@
   {/if}
 
   {#if checkSlotExists('logo')}
-    <a class="vtmn-navbar_logo" href={logoHref}>
```
Comment (Tlahey) [inline packages/sources/svelte/src/components/navigation/VtmnNavbar/VtmnNavbar.svelte:42]: Also change it on : 
- packages/showcases/css/stories/components/navigation/navbar/examples/overview.html
- packages/sources/vue/src/components/navigation/VtmnNavbar/VtmnNavbar.vue

React is already a `<div>`

**Comment by lauthieb** [comment]
Hi @Mig-l, thanks for your help.

In order to avoid several bump with a major version, I propose to merge your PR to the PR https://github.com/Decathlon/vitamin-web/pull/1425 of @Tlahey hosted in `feat/vtmnNavbarLink-enhancement-ssr` git branch. That's why I changed the target branch from `main` to `feat/vtmnNavbarLink-enhancement-ssr`.`

Can you please apply its suggestions & also do the same job for Vue & React implementations?

Thank you!

File: packages/sources/svelte/src/components/navigation/VtmnNavbar/VtmnNavbar.svelte (hunk 2 above)
Comment (Mig-l) [reply packages/sources/svelte/src/components/navigation/VtmnNavbar/VtmnNavbar.svelte:42]: Done

File: packages/showcases/core/csf/components/navigation/navbar.csf.js [hunk 3]
```diff
@@ -5,11 +5,4 @@ export const parameters = {
...
 
-export const argTypes = {
-  logoHref: {
-    type: { name: 'string', required: false },
-    description: 'Logo anchor',
-    defaultValue: '#',
-    control: { type: 'text' },
-  },
-};
+export const argTypes = {};
```
Comment (Tlahey) [inline packages/showcases/core/csf/components/navigation/navbar.csf.js:8]: can be removed

Comment (Mig-l) [reply packages/showcases/core/csf/components/navigation/navbar.csf.js]: Done

**Review by thibault-mahe — APPROVED** [review]

**Comment by thibault-mahe** [comment]
It's ok for me, thanks a lot @Mig-l :)

**Pull request merged**